src/Files.App/Views/MainPage.xaml

```diff
@@ -280,32 +268,25 @@
 						ResizeBehavior="BasedOnAlignment"
 						Style="{StaticResource DefaultGridSplitterStyle}" />
 
-					<!--  Preview Pane  -->
-					<Border
-						x:Name="InfoPaneContainer"
+					<!--  Info Pane (Lazy Load)  -->
+					<uc:InfoPane
+						x:Name="InfoPane"
```

Comment:
Are there any visual states that need updating to reflect this change?

Reply:
I don't think so.
I already updated in Main page.xaml.cs